Packs/CortexResponseAndRemediation/Triggers/Trigger_-_Suspicious_certutil_command_line.json

```diff
@@ -0,0 +1,19 @@
+{
+  "trigger_id": "d30b757218c4a36e4b94d8033cf55785",
+  "playbook_id": "Suspicious certutil command line",
+  "suggestion_reason": "Recommended for the 'Suspicious certutil command line' alert",
+  "description": "This trigger is responsible for handling 'Suspicious certutil command line' alert",
```

Comment:
This trigger is responsible for handling 'Suspicious certutil command line' alerts",